nautobot/docs/user-guide/core-data-model/dcim/interface.md

```diff
@@ -27,11 +23,7 @@ IP addresses can be assigned to interfaces. VLANs can also be assigned to each i
 !!! note
     Although devices and virtual machines both can have interfaces, a separate model is used for each. Thus, device interfaces have some properties that are not present on virtual machine interfaces and vice versa.
 
-+++ 1.4.5
-    The fields `created` and `last_updated` were added to all device component models. If you upgraded from Nautobot 1.4.4 or earlier, the values for these fields will default to `None` (null).
 
-+++ 1.6.0
-    Interfaces can now be assigned to an [Interface Redundancy Group](./interfaceredundancygroup.md) to represent redundancy protocols such as HSRP or VRRP.
```

Comment:
Maybe keep this paragraph as a feature note?

Reply:
@glennmatthews just to confirm, are you talking about keeping only the following:
`Interfaces can now be assigned to an [Interface Redundancy Group](./interfaceredundancygroup.md) to represent redundancy protocols such as HSRP or VRRP.`

If so, do you think it makes sense to include it in the previous paragraph 
https://github.com/nautobot/nautobot/pull/6903/files/262e78bf6b3b04ded9b6ab2751391ce0522612fd#diff-f8d851681c146a7e766841168882aeaecc18cf46a40ddec0340c628741ddb535L20

which is already talking about interfaces? If you want to keep as feature note, that works too. I am going to hold on this change until I get confirmation.

Reply:
Sounds good to me.